provisioning/Samples/device/SymmetricKeyWithOperationalCertificateSample/readme.md

```diff
@@ -1,7 +1,7 @@
 # Using Azure IoT Device Provisioning Certificate Signing Requests
 
 - Device Provisioning Service (DPS) can be configured to receive Certificate Signing Requests (CSRs) from IoT devices as a part of the DPS registration process. DPS will then forward the CSR on to the Certificate Authority (CA) linked to your DPS instance. 
-- The CA will sign and return an X.509 device identity certificate (aka client certificate) to DPS. We refer to this as an operational certificate. DPS will register the device and operational client certificate thumbprint in IoT hub and return the certificate to the IoT device. The IoT device can then use the operational certificate to authenticate with IoT hub. 
+- The CA will sign and return an X.509 device identity certificate (aka client certificate) to DPS. We refer to this as an operational certificate. DPS will register the device and operational client certificate thumbprint in IoT hub and return the certificate to the IoT device. The IoT device can then use the returned operational certificate along with the private key information to authenticate with IoT Hub.
```

Comment:
Previous case was correct. They don't want us capitalizing the H in hub when referring to an instance.

```suggestion
- The CA will sign and return an X.509 device identity certificate (aka client certificate) to DPS. We refer to this as an operational certificate. DPS will register the device and operational client certificate thumbprint in IoT hub and return the certificate to the IoT device. The IoT device can then use the returned operational certificate along with the private key information to authenticate with IoT hub.
```